pytest_splunk_addon/standard_lib/data_models/Alerts.json

```diff
@@ -64,7 +64,9 @@
                     "alarm",
                     "alert",
                     "event",
-                    "task"
+                    "task",
+                    "warning",
+                    "unknown"
```

Comment:
"unknown" indicates a default value when testing we should always find one of the other values "unknown" itself is a bug

Reply:
@rfaircloth-splunk Done.